addons/mail/static/src/core_ui/thread.xml

```diff
@@ -4,8 +4,9 @@
 <t t-name="mail.Thread" owl="1">
     <t t-if="env.inChatter" t-call="mail.Thread.jumpPresent"/> <!-- chatter has its own scrollable, this ensures proper sticky showing -->
     <div class="o-mail-Thread flex-grow-1" t-att-class="{ 'overflow-auto': props.hasScrollAdjust, 'pb-4': !state.showJumpPresent }" t-ref="messages">
+        <t t-name="messageListBefore"/>
```

Comment:
Isn't `t-name` supposed to be used to define the name of a template, top-level?
Just name here would work? Or use "before content" as the following node is already named?

Reply:
Indeed, prefer `t-ref`/`name` (I'm indifferent in whichever we pick, but certainly not `t-name`)

Reply:
used `name="content"` as suggested